### PR TITLE
VWO documentation update

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> VWO documentation revamp
> Added 1 new FAQ

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/destinations/streaming-destinations/vwo-beta-visual-website-optimizer.mdx [hunk 1]
```diff
@@ -16,64 +14,130 @@ RudderStack supports integration with VWO that allows you to send relevant event
...
+<div class="infoBlock">
+For more information, refer to the <a href="https://help.vwo.com/hc/en-us/articles/360008469173-How-to-find-your-account-ID">VWO Knowledge Base</a>.
+</div>
 
+### How can I prevent the VWO SDK from loading on my website?
+
+To disable the VWO SDK and prevent it from loading, set `loadIntegration` to `false` in the event's `integrations` object, as shown:
+
+```javascript
+rudderanalytics.track(
```
Comment (yashasvibajpai) [inline docs/destinations/streaming-destinations/vwo-beta-visual-website-optimizer.mdx:132]:
```suggestion
rudderanalytics.track(
  "Experiment Viewed", {
    revenue: 30,
    currency: "USD",
  }, {
  integrations: {
    VWO: {
      loadIntegration: false
    }
   },
  }
);
```

Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/vwo-beta-visual-website-optimizer.mdx:132]: Thanks @yashasvibajpai incorporated this snippet.

**Comment by ameypv-rudder** [comment]
@yashasvibajpai @utsabc team, we don't have a Beta tag on the webapp for this destination anymore. Should we remove the tag from the doc as well?

**Comment by yashasvibajpai** [comment]
> @yashasvibajpai @utsabc team, we don't have a Beta tag on the webapp for this destination anymore. Should we remove the tag from the doc as well?

Yes as we usually utilize beta tag for newly released destinations that have some scope of change some time soon.

**Pull request closed without merging**